README.rst

```diff
@@ -42,19 +63,45 @@ Add the AutoExtract downloader middleware in the settings file::
 
 Note that this should be the last downloader middleware to be executed.
 
+The providers
+-------------
 
-Usage
-=====
+Another way of consuming AutoExtract API is using the Page Objects pattern
+proposed by the `web-poet`_ library and implemented by `scrapy-poet`_.
 
-The middleware is opt-in and can be explicitly enabled per request,
-with the ``{'autoextract': {'enabled': True}}`` request meta.
-All the options below can be set either in the project settings file,
-or just for specific spiders, in the ``custom_settings`` dict.
+Page Objects their returned Items are defined by the `autoextract-poet`_
+library.
+
+Within the spider, consuming the AutoExtract result is as easy as::
+
+    from autoextract_poet.page_inputs import AutoExtractArticleData
+
+    def parse(self, response, article: AutoExtractArticleData):
+        yield article.to_item()
```

Comment:
I'd add the spider code around it as well.
It would be 5 other lines, but would make it easier to link to a request and the parse and also making it explicit that a single request is being downloaded twice